config.py

```diff
@@ -20,7 +20,13 @@ def __getattr__(self, attrname):
             warnings.warn("The `slack_name` key in %s is deprecated in favor of the `SLACK_NAME` environment variable" %
                           self.config_fname, DeprecationWarning)
 
-        return self.config[attrname]
+        if os.getenv(attrname.upper()) is not None:
+            return os.getenv(attrname.upper()).split(',') if ',' in os.getenv(attrname.upper()) else os.getenv(attrname.upper())
+
+        try:
```

Comment:
I think I tried 4 different ways to access attributes (yay Python) which all threw accessor exceptions when you pulled an attribute which didn't exist. What you've suggested works perfectly for what I was trying to achieve with this try/catch. I'll update with those changes.